buildme.pl

```diff
@@ -17,29 +17,50 @@
 my $sourceDirsToExclude = ".svn .git tests slimp3 squeezebox /softsqueeze tools ext/source ext-all-debug.js build"; 
 my $revisionTextFile = "server/revision.txt";
 my $revision;
-my $myVersion = "0.0.3";
+my $myVersion = "0.0.4";
 my $defaultDestName = "logitechmediaserver";
 my $defaultReleaseType = "nightly";
 
 ## Windows Specific Stuff
 my $windowsPerlDir = "C:\\perl";
 my $windowsPerlPath = "$windowsPerlDir\\bin\\perl.exe";
 
+my $icuBigEnd = "icudt[0-9][0-9]b.dat ";
+my $icuLittleEnd = "icudt[0-9][0-9]l.dat ";
+my $icuNew = "icudt[5-9][0-9][bl].dat ";
+my $freebsdDirs = "i386-freebsd-64int ";
+my $solarisDirs = "i86pc-solaris-thread-multi-64int ";
+my $mainLinuxDirs = "i386-linux x86_64-linux ";
+my $armHFLinuxDirs = "armhf-linux ";
+my $arm64LinuxDirs = "aarch64-linux ";
+my $armLinuxDirs = "arm-linux aarch-linux " . $armHFLinuxDirs . $arm64LinuxDirs;
+my $sparcLinuxDirs = "sparc-linux sparc-unknown-linux ";
+my $ppcLinuxDirs = "powerpc-linux ";
+my $linuxDirs = $armLinuxDirs . $sparcLinuxDirs . $mainLinuxDirs . $ppcLinuxDirs;
+my $win32Dirs = "MSWin32-x86 ";
```

Comment:
That should be MSWin32-x86-multi-thread.

Reply:
Since it's a grep pattern, won't it find what it needs?